src/webserver.go

```diff
@@ -346,6 +346,15 @@ func Threadfin(w http.ResponseWriter, r *http.Request) {
 		requestType = "m3u"
 		groupTitle = r.URL.Query().Get("group-title")
 
+                // Check auth and if not present exit. Same as in Line 388
```

Comment:
Should read as: Check auth and if is not valid combination exits... Same as in Line 388